src/main/java/frc/robot/elevator/ElevatorIOSim.java

```diff
@@ -1,62 +1,80 @@
 package frc.robot.elevator;
 
+import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.math.controller.ElevatorFeedforward;
 import edu.wpi.first.math.controller.ProfiledPIDController;
 import edu.wpi.first.math.system.plant.DCMotor;
 import edu.wpi.first.math.trajectory.TrapezoidProfile.Constraints;
 import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.simulation.ElevatorSim;
 
 public class ElevatorIOSim implements ElevatorIO {
-  private final ElevatorSim sim =
-      // TODO elevator sim
+  private final ElevatorSim physicsSim =
       new ElevatorSim(
-          DCMotor.getKrakenX60Foc(1),
+          DCMotor.getKrakenX60Foc(2),
           ElevatorSubsystem.GEAR_RATIO,
-          Units.lbsToKilograms(7.0 + (3.25 / 2)),
-          ElevatorSubsystem.DRUM_RADIUS_METERS,
+          // Add half of first stage mass bc its on a 2:1 ratio compared to carriage
+          // First stage weighs 3.345 lbs
+          // Carriage weighs 8.863
+          // Arm weighs 5.625
+          Units.lbsToKilograms((3.345 / 2) + 8.863 + 5.625),
+          ElevatorSubsystem.SPROCKET_DIAMETER_METERS,
```

Comment:
```suggestion
          ElevatorSubsystem.SPROCKET_DIAMETER_METERS / 2.0,
```
it takes radius instead of diameter